Corona-Warn-App/src/main/java/de/rki/coronawarnapp/qrcode/ui/QrCodeScannerFragment.kt

```diff
@@ -117,7 +118,8 @@ class QrCodeScannerFragment : Fragment(R.layout.fragment_qrcode_scanner), AutoIn
     private fun onDccTicketingResult(scannerResult: DccTicketingResult) {
         when (scannerResult) {
             is DccTicketingResult.ConsentI -> {
-                // TODO
+                // TODO navigate to Consent I
+                Toast.makeText(requireContext(), "Consent I :)", Toast.LENGTH_LONG).show()
```

Comment:
@AlexanderAlferov  `DccTicketingResult.ConsentI` is not providing the `TransactionContext` that you can use to open consent I